src/View/JsonView.php

```diff
@@ -147,7 +145,12 @@ protected function _serialize($serialize)
             $jsonOptions |= JSON_PRETTY_PRINT;
         }
 
-        return json_encode($data, $jsonOptions);
+        $return = json_encode($data, $jsonOptions);
+        if ($return === false) {
+            throw new RuntimeException(json_last_error_msg(), json_last_error());
```

Comment:
Should this be the new exception type? It would save you a try/catch in the parent.

Reply:
The try/catch in parent would still be needed to catch potential XmlException from XmlView and throwing SerializationFailureException instead.

Throwing this RuntimeException is necessary only because JsonException is available since PHP 7.3 :(

Reply:
The try/catch in `SerializedView` is also necessary for non core view classes extending `SerializedView`, for .e.g. `CsvView` of `friendsofcake/cakephp-csvview`.